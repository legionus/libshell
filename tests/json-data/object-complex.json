{
	"foo": {
		"asdf": "qwerty",
		"hooah": "HOOAH"
	},
	"bar": {
		"asdf": "qwerty",
		"hooah": "HOOAH"
	},
	"baz": {
		"asdf": "qwerty",
		"hooah": "HOOAH"
	}
}
