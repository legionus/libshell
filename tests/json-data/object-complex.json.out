/foo/asdf string qwerty
/foo/hooah string HOOAH
/bar/asdf string qwerty
/bar/hooah string HOOAH
/baz/asdf string qwerty
/baz/hooah string HOOAH
